modules/ROOT/pages/_partials/include-nav-steps-team-maintainer.adoc

```diff
@@ -0,0 +1,3 @@
+. Log in to Anypoint Platform using an account that is a team maintainer of the team you want to modify.
+. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
+. In the Access Management navigation menu, click *Teams*.
```

Comment:
If the user is _only_ a team maintainer, then they will only have access to `Teams` and will automatically be redirected there. Not sure how to capture that, or if necessary